File: 1768_merge_strings_alternately/merge_strings_alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int i=0, j=0;
        std::string res = "";
        while(i < word1.length() && j < word2.length())
        {
            res.push_back(word1[i]);
            res.push_back(word2[j]);
            i++;
            j++;
        }
        if(word1.length() > word2.length())
        {
            for(; i < word1.length(); i++)
                res.push_back(word1[i]);
        }
        else
        {
            for(; j < word2.length(); j++)
                res.push_back(word2[j]);
        }
        return res;
    }
};

File: 1768_merge_strings_alternately/merge_strings_alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merged_str = str()
        i = 0
        j = 0
        while i < len(word1) and j < len(word2):
            merged_str += word1[i]
            merged_str += word2[j]
            i += 1
            j += 1

        if len(word1) > len(word2):
            merged_str += str(word1[i:])

        else:
            merged_str += str(word2[j:])

        return merged_str